app/highlights/components/newHighlightList.js

```diff
@@ -0,0 +1,25 @@
+const h = require('vdux/element').default
+
+const { map } = require('lodash')
+
+const actions = require('../actions')
+
+module.exports = {
+  render,
+  onCreate
+}
+
+function render ({ props, children }) {
+  return h('div', {id: 'newHighlightList'}, map(children, (child, childKey) => {
+    return h(child, {}, [])
+  }),
+  h('button', {onClick: addHighlight }, 'Add new highlight'))
+}
+function addHighlight() {
+  //How to add a component I don't know about?
```

Comment:
Makes sense, thanks. Just curious how I could do it. Will come find you in person